### DISCUSSION TITLE
Unable to verify secondary email address to account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49358] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi all!
Kindly request help to resolve a possible bug with adding a second email address to my account.
The problem is that I cannot get that second email address verified.

I'm in the process of configuring my GitHub account. I've added a primary email address. Now I want to change that address. 
It's not possible to just delete the primary address, so I add a second address: mkla1@gmail.com, and click the "Resend verification email" link next to the label "Unverified".
In my email inbox, I find the verification email but there is no verification link to click:

```
Hey mkla1!

An email address ([mkla1@gmail.com](mailto:mkla1@gmail.com)) was added to your account. Visit https://github.com/settings/emails to review the email addresses currently associated with your account.

To see this and other security events for your account, visit https://github.com/settings/security-log

If you run into problems, please contact support by visiting https://github.com/contact

Thanks,
The GitHub Team
```

As per the instruction in step 4 at...:

https://docs.github.com/en/enterprise-cloud@latest/get-started/signing-up-for-github/verifying-your-email-address

...it reads:
"GitHub will send you an email with a link in it. After you click that link, you'll be taken to your GitHub dashboard and see a confirmation banner. "

I can find no such confirmation link in the email.
Thanks in advance for any help or suggestions

Mikael Larsson

**Comment by ghost** [comment]
Misunderstanding of function on my part.

**Reply by adityaraj-h** [reply]
what was the problem? Did you find a solution? I am facing the same problem
